11 - Custom Video Player/scripts.js

```diff
@@ -0,0 +1,103 @@
+class CustomVideoPlayer {
+    constructor(selector) {
+        this.player = document.querySelector(selector);
+        this.video = this.player.querySelector('.viewer');
+        this.progress = this.player.querySelector('.progress');
+        this.progressBar = this.player.querySelector('.progress__filled');
+        this.toggle = this.player.querySelector('.toggle');
+        this.skipButtons = this.player.querySelectorAll('[data-skip]');
+        this.ranges = this.player.querySelectorAll('.player__slider');
+        this.fullScreenToggle = this.player.querySelector('.fullScreenToggle');
+        this.mouseDown = false;
+    };
+
+    playPause = () => {
+        if (this.video.paused) this.video.play();
```

Comment:
interesting indentation style

Reply:
You mean the closing curly probably. Fixed 😄